Sprint-2/debug/2.js

```diff
@@ -2,8 +2,8 @@
 
 const num = 103;
 
-function getLastDigit() {
-  return num.toString().slice(-1);
+function getLastDigit(number) {
```

Comment:
This works fine although `number` is very close to the Javascript function `Number` so might be a bit confusing for someone to read, how about using `num` as the parameter name instead?

Reply:
Also, is the `num` declaration on line 3 still needed? If not, it's best to remove it.

Reply:
thanks. I applied your suggestions